src/AbpHelper.Core/Steps/StepWithOption.cs

```diff
@@ -109,12 +109,12 @@ public WorkflowExpression<bool> CommandOption
                     matchedSln = true;
                 }
 
-                var result = searchFunc(directory, pattern, SearchOption.TopDirectoryOnly).FirstOrDefault();
-                if (!result.IsNullOrWhiteSpace())
+                var result = searchFunc(directory, pattern, SearchOption.TopDirectoryOnly);
+                if (result.Any())
```

Comment:
`if (result is not null)`

Reply:
![image](https://github.com/EasyAbp/AbpHelper.CLI/assets/34080651/2d50e4a6-d649-4848-98d0-0ab876d1154f)

the `Directory.EnumerateFiles`  return `IEnumerable<string>` .`result is not null` will alway return `ture`

Reply:
![image](https://github.com/EasyAbp/AbpHelper.CLI/assets/34080651/74b6d113-a465-468e-b55d-f382dd8897c2)

Reply:
Yep, `Directory.EnumerateFiles` will never return null, but `searchFunc` will. So please check the null value.

Reply:
`result is not null` will cause a warning in the IDE if the nullable is enabled for this project.

Now I think this "if" line can just be removed.

Reply:
the `FindInDirectoryInQueue()` maybe have same probleam.
`line:149 var result = searchFunc(directory, pattern, SearchOption.TopDirectoryOnly).FirstOrDefault();`

Reply:
```
private static string? FindInDirectoryInQueue(Queue<string> queue, string pattern,
            IReadOnlySet<string> actualExcluded, Func<string, string, SearchOption, IEnumerable<string>> searchFunc,
            bool includeSubModules = false)
        {
            var matchedSln = false;
            while (queue.TryDequeue(out var directory))
            {
                if (!includeSubModules &&
                    null != searchFunc(directory, "*.sln", SearchOption.TopDirectoryOnly).FirstOrDefault())
                {
                    if (matchedSln)
                    {
                        continue;
                    }

                    matchedSln = true;
                }

                //var result = searchFunc(directory, pattern, SearchOption.TopDirectoryOnly).FirstOrDefault();
                //if (!result.IsNullOrWhiteSpace())
                //{
                //    return result;
                //}
                
                var result = searchFunc(directory, pattern, SearchOption.TopDirectoryOnly);
                if (result != null)
                {
                    return result.FirstOrDefault();
                }

                foreach (var d in Directory.EnumerateDirectories(directory))
                {
                    if (actualExcluded.Contains(d))
                    {
                        continue;
                    }

                    queue.Enqueue(d);
                }
            }

            return null;
        }
```